packages/@webex/plugin-meetings/src/multistream/remoteMediaGroup.ts

```diff
@@ -210,28 +212,47 @@ export class RemoteMediaGroup {
     }
   }
 
-  private sendActiveSpeakerMediaRequest(commit: boolean) {
+  public sendActiveSpeakerMediaRequest(commit: boolean) {
```

Comment:
This method should remain private, see my other comment about creating setNamedMediaGroup() instead.

Reply:
Done